app/controllers/application_controller.rb

```diff
@@ -1,5 +1,60 @@
 class ApplicationController < ActionController::Base
```

Comment:
Right... I actually attempted to move all that stuff from `initializers/controllers.rb` but oh... after I did, a lot things broke and I couldn't figure out why until I realized bit—by—bit that it was only working because of that magic.

So, I don't know, what do you want me to do? My goal is to remove as much as possible from controllers.rb and eventually all of it but it'll only happen after FinePrint is fixed.

Reply:
Let's do this in a separate PR.

Reply:
Ok. Note to self: try not to forget.